### PR TITLE
Finish admin manages users

### PR DESCRIPTION
**Pull request by binhthanhnguyen**

### CODE REVIEW

File: app/views/admin/users/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+          <td><%= user.id %></td>
+          <td><%= user.name %></td>
+          <td><%= user.email %></td>
+          <td><%= user.created_at.strftime("%Y/%m/%d") %></td>
+          <td><%= user.updated_at.strftime("%B %e %Y") %></td>
+        </tr>
+      <% end %>
+    </table>
+  </aside>
+</div>
```
Comment (ThuBM) [inline app/views/admin/users/index.html.erb]: new line

File: app/controllers/admin/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class Admin::UsersController < ApplicationController
+  def index
+    @users = User.all
```
Comment (dieunb) [inline app/controllers/admin/users_controller.rb]: paginate

File: app/models/lesson.rb [hunk 3]
```diff
@@ -18,6 +18,10 @@ def num_correct_answers
     answers.select{|answer| answer.option.correct? unless answer.option.nil?}.count
   end
 
+  def learned_words
```
Comment (dieunb) [inline app/models/lesson.rb]: learned_words_count

File: app/views/lessons/show.html.erb [hunk 4]
```diff
@@ -27,4 +27,13 @@
     <hr>
   <% end %>
   <%= f.submit "Submit", class: "btn btn-primary" %>
-<% end %>
\ No newline at end of file
+<% end %>
+
+<script type="text/javascript">
```
Comment (dieunb) [inline app/views/lessons/show.html.erb]: move javascipt code to js file

File: app/views/lessons/show.html.erb [hunk 5]
```diff
@@ -27,4 +27,13 @@
...
+  var countChecked = function() {
+    var n = $( "input:checked" ).length;
+    $( "#lesson_stats" ).html( n + "/<%= @lesson.words.count %>");
+  };
+  countChecked();
+  $("input").on("click",countChecked );
+</script>
```
Comment (ThuBM) [inline app/views/lessons/show.html.erb]: new line

File: app/controllers/admin/users_controller.rb (hunk 2 above)
Comment (binhthanhnguyen) [inline app/controllers/admin/users_controller.rb]: here i display user by table, so i think i don't need to use paginate? is that right?

**Pull request merged**